src/components/confirm-dialog/confirm-dialog-content.html

```diff
@@ -0,0 +1,11 @@
+<header class="mdc-dialog__header">
+  <h2 class="mdc-dialog__header__title">
+    Delete
+    <%= type %>
+  </h2>
+</header>
+<section class="mdc-dialog__body">
+  <p>Are you sure you want to delete
+    <%= type %>
```

Comment:
why do you need these placeholders for ejs template? you can declare just 2 `<span>`s,  give them class names, query them in component and output value there